fe/fe-core/src/main/java/com/starrocks/catalog/exception/CatalogException.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.catalog.exception;
+
+import com.starrocks.common.InternalErrorCode;
+
+/**
+ * Exception for meta info is null, like db table partition tablet replica job
+ */
+public class CatalogException extends RuntimeException {
```

Comment:
We alreay have `StarRocksConnectorException`, I think you could reuse this exception

Reply:
I thought `CatalogException` is required as it is Catalog exception, `StarRocksConnectorException` represents a connector level exception, which is much wider.
WDYT?

Reply:
I think using  StarRocksConnectorException   is sufficient. We have already thrown  StarRocksConnectorException   in many other places. Changing it to   CatalogException  might involve additional modifications.

Reply:
I modified it to use `StarRocksConnectorException`

Reply:
You can just delete this file